content/redirect.js

```diff
@@ -94,6 +98,13 @@ function load(files = null, verbose = false) {
       .readdirSync(CONTENT_ROOT)
       .map((n) => path.join(CONTENT_ROOT, n))
       .filter((filepath) => fs.statSync(filepath).isDirectory());
+    if (CONTENT_TRANSLATED_ROOT && fs.existsSync(CONTENT_TRANSLATED_ROOT)) {
```

Comment:
No need to check if it exists. See https://github.com/mdn/yari/blob/c1a5159d2ce16c0062f087285ebbeebb6bf8b69d/content/constants.js#L38-L39